<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="application-name" content="SJTU-BioX-Shanghai">
    <meta name="description" content="History of SJTU-BioX-Shanghai, the iGEM team of Shanghai Jiao Tong University">
    <meta name="keywords" content="SJTU,Bio-X,synthetic biology,biology,iGEM,iGEM competition,Shanghai Jiao Tong University">
    <meta name="Author" Content="Wazon,Wazons@163.com">
    <title>History - SJTU-BioX-Shanghai</title>
    <link rel="icon" href="./favicon.ico" type="image/x-icon">
    <link rel="shortcut icon" href="./favicon.ico" type=”image/x-icon”>
    <link href="./css/bootstrap.min.css" rel="stylesheet">
    <!-- Material Design Bootstrap -->
    <link href="./css/mdb.min.css" rel="stylesheet">
    <link href="./css/font-awesome.min.css" rel="stylesheet">
    <link href="./css/site.css" rel="stylesheet">
    <!--[if lt IE 10]>
        <center>您使用的是过时的浏览器，浏览本网页可能会有异常。</center>
        <center>You are using an outdated browser and this page may not display properly.</center>
    <![endif]-->
    <style>
        .navbar {
            background-color: #951016;
            z-index: 1;
        }
        
        .onesection p {
            line-height: 1.5em;
            margin-top: 0.5em;
            margin-bottom: 0rem;
        }
    </style>
    <script src="./js/analysis.js"></script>
</head>

<body class="grey lighten-5" data-spy="scroll" data-target="#navbar-right" data-offset="150">

    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
        <div class="container">
            <a class="navbar-brand" href="#"><img src="./img/png/logo.png" height="55" /></a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav1" aria-controls="navbarNav1" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarNav1">
                <ul class="navbar-nav mr-auto">
                    <li class="nav-item"><a href="./index-en.html" class="nav-link">Home</a></li>
                    <li class="nav-item"><a href="./intro-en.html" class="nav-link">Introduction</a></li>
                    <li class="nav-item active"><a href="./history-en.html" class="nav-link">History<span class="sr-only">(current)</span></a></li>
                    <li class="nav-item"><a href="./practices-en.html" class="nav-link">Practices</a></li>
                    <li class="nav-item"><a href="./outcomes-en.html" class="nav-link">Outcomes</a></li>
                </ul>
                <ul class="navbar-nav ml-auto">
                    <li class="nav-item dropdown"><a href="./history.html" class="nav-link">简体中文</a></li>
                </ul>
            </div>
        </div>
    </nav>

    <div class="container">
        <div class="row middle-row align-items-top justify-content-center">

            <div class="col-12 col-md-8 base-responsive " style="margin-bottom:3rem">
                <div class="section-helper" id="section2">
                    <div style="height:50px"></div>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2009 E.coli the Napper</h3>
                    <p>It is universally acknowledged that bioclock works as a circadian regulator in most eukaryotic multicellular species. This mechanism controls higher plants’ blossom time, brings insects into metamorphosis, and also wakes us up every
                        day. Then comes up the crazy idea: Why cannot prokaryotes live with a bioclock? Hence, we constructed our bacteria bioclock by utilizing the toxin-antitoxin system (TA system), which forms an oscillator between two physiological
                        states--dormancy and activity.
                    </p>
                    <p>
                        <a target="_blank" href="http://2009.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section3">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2010 Synthetic-biological Approaches to Osteoarthritis</h3>
                    <p>Osteoarthritis (OA) is a chronical disease in which joint matrix is degraded and chondrocytes undergo disorderd and hypertrophic differentiation, symptoms including joint pain, tenderness and stiffness. The National Arthritis data
                        Workgroup estimates the prevalence of OA in the United States as 26.9 million in 2005; this indicates a rise of nearly 30% over the course of the previous 10 years. Current treatments are focused on symptomatic relief but they
                        lack efﬁcacy to control the progression of this disease which is a leading cause of disability. Therefore, correct diagnosis and therapy is critical, since appropriate therapy influences not only joint function and quality of life,
                        but can also prevent relevant end-organ damage.</p>
                    <p> We proposed two synthetic-biological approaches to OA, one with a eukaryotic genetic circuit and another prokayotic. Both circuits are composed of three systems: “Detector”, “Actuator” and “Supervisor”. As for Detector, we built tissue-specific
                        promoters in the eukaryotic circuit, while inflammation factors are employed as signals of OA in the prokaryotic circuit. The same Actuator shared by two circuits generates proteins col2a1, which replenishes the degraded matrix,
                        and oct4, which reverses the disordered differentiation. The eukaryotic Supervisor part has an original design in which a photo-sensitive cation channel crosstalks with certain cellular signaling pathways, resulting in the light-controlled
                        expression of col2a1 and oct4; in the counterpart of prokaryotic circuit, both injected inducers and over-population lead the engineered bacteria to suicide, thus attenuating possible side effects.

                    </p>
                    <p>
                        <a target="_blank" href="http://2010.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section4">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2011 Codon Switch Controlling Protein Biosynthesis</h3>
                    <p>This year we designed a set of Codon-Switches that regulate target protein biosynthesis (translation). In our Rare-Codon Switch, the translation of the protein can be finely turned up/down with the control of rare tRNA amount, aaRS
                        that charges the rare tRNA and rare codons. Besides, our device can be made into switches that can be turned on/off without background noise in two ways. One is to use stop codon as the controlling element, the Stop-Codon Switch.
                        The other is to use any codon but the original start codon to initiate translation, the Initial-Codon Switch. Our design has expanded the regulating tools for synthetic biology and introduced brand-new methods for protein function
                        analysis.
                    </p>
                    <p>
                        <a target="_blank" href="http://2011.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section5">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2012 Membrane Magic</h3>
                    <p>This year, SJTU-BioX-Shanghai iGEM team has build a "factory" on E.coli's membrane, where enzyme assemblies can be manipulated so that biochemical reactions can be accelerated and further controlled. </p>
                    <p>We aimed at constructing a set of protein assemblies on E.coli inner membrane as scaffold carrying various enzymes. Distinct from previous synthetic scaffold system, our device limits the reaction space to a two-dimensional surface.
                        In such system, the membrane functions as an extensive scaffold for proteins to anchor without limitation of scaffold amount. Membrane as scaffold also has privilege in receiving external and internal regulating signals. Based
                        on Membrane Scaffold, we built two universal devices: Membrane Accelerator and Membrane Rudder.</p>
                    <p> In Membrane Accelerator device, by gathering enzymes on membrane, production of fatty acid was enhanced by more than 24 fold, which has a promising application prospect in biofuel production. We also proposed a new direction for application
                        of scaffold-based accelerator: Biodegradation. </p>
                    <p>In Membrane Rudder device, we changed the direction of Violacein synthetic pathway through external signal.
                    </p>
                    <p>
                        <a target="_blank" href="http://2012.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section6">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2013 Metabolic Gear Box</h3>
                    <p> How to regulate an entire metabolic pathway in vivo, delicately, accurately and conveniently, simultaneously controlling the expression of several genes? </p>
                    <p>And how to optimize metabolic fluxes so as to maximize desired products?
                    </p>
                    <p> For decades, these questions have haunted “bioengineers”(especially synthetic biologists), who would like to have certain metabolites produced in living cells. The difficulty is again raised up when target genes endogenously reside
                        on the genome, or have been implemented into the genome.
                    </p>
                    <p>So this year, our team, SJTU-BioX-Shanghai, is dedicated to solve the problem. We offer our product, the METABOLIC GEAR BOX, as a versatile BLACK BOX that can be conveniently applied in the optimization of whatever pathways.
                    </p>
                    <p>Inside the BOX, we integrate Luminous System, Light Sensors and CRISPRi. Luminous system and light sensors are used to address the problem of multiple gene regulation, while CRISPRi addresses the problem of genomic regulation.
                    </p>
                    <p>
                        <a target="_blank" href="http://2013.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section7">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2014 Crown</h3>
                    <p>This year, our project intends to achieve protein polymerization with the help of TAL effector (Tale Transcription Activator–like Effector), a DNA-binding protein, which can recognize a specific nucleotide sequence. With circular DNA(plasmid)
                        as the connection medium, expressed proteins in the host can form a complex selectively, so that a variety of enzyme combinations can be used to complete different tasks. Compared to traditional methods, the advantages of the project
                        lie in selective polymerization of proteins and the ability to mediate polymerization between more proteins.
                    </p>
                    <p>
                        <a target="_blank" href="http://2014.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section8">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2015 Cyano Pac-man</h3>
                    <p>Reverse Osmosis, the mostly-used desalination technology, uses artificial membrane to extract salt. Have you ever considered biological membrane? We use cyanobacteria as our desalination organism, the light-driven chloride pump as
                        our desalination driver, dark-sensing promoter as our desalination controller, to establish a primary biodesalination system. We managed to decrease the concentration of sodium chloride by 20% and more optimizations may generate
                        a greater degree of reduction.
                    </p>
                    <p>
                        <a target="_blank" href="http://2015.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section9">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2016 Vesatile Yeast Interface - for disease diagnosis</h3>
                    <p>We managed to engineer Saccharomyces cerevisiae, a yeast strain with great abundance of background information and simple genetic manipulations, to make it a promising choice for individual to make early diagnosis domestically with
                        low cost. The most exciting part of our story is that our yeast biosensor can not only serve as a chassis for an incredibly large variety of disease biomarkers, just like a versatile interface in terms of computer engineering,
                        but also response in a real-time manner which allows users to get their results within 10 minutes.
                    </p>
                    <p>
                        We constructed the biosensor for diagnosis of two diseases, diabetes and phaeochromocytoma, using urine sugar and epinephrine as their biomarkers. Three major types of elements to genetically modify the yeast Saccharomyces cerevisiae: The first type of
                        elements is promoter, they are used to control the expression of other elements in yeast; the second type of elements is reporter protein, Nano-lantern(cAMP-1.6), which allows detection of cellular response to extracellular ligands
                        within 10 minutes; the third type of elements is G protein coupled receptor, ADRB2, which allows detection of the disease biomarker of phaeochromocytoma, excreted epinephrine.
                    </p>
                    <p>
                        <a target="_blank" href="http://2016.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                <div class="section-helper" id="section10">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2017 Palette</h3>
                    <p>This year's project is named Palette - A Universal Multifactorial Visualized Detection System, which applies synthetic biology. At present, there are various kinds of multifactorial systems in our life, such as a diversity of heavy
                        metal pollution in water or soil, a variety of additives in food, multiple disease factors in human blood or body fluid and so on. Fast and easy factor detection is getting more and more urgent and important. </p>
                    <p>We adopted a new gene transcription control technology and chromoprotein to build a portable biometric system. Combined with cell phone and APP analysis, we are able to colorize different factors and achieve quantitative detection
                        of multiple factors through the color superposition. The project innovatively adopted the STAR (Small Transcriptional-Activating RNA) technology to control the expression of the gene, and our team member Zhuoyang Chen developed
                        a new STAR based on the existing literature and significantly improved Detection efficiency. The experiment introduced the STAR gene with the orthogonality among each other in Escherichia coli to control the expression of different
                        pigment genes separately. The visibility of the naked eyes and pigment protein make it possible to achievethe multi-factor colorimetric detection, breaking the limitation of the traditional fluorescence detection which needs to
                        rely on the limitations of laboratory equipment.</p>
                    <p> With the help from Department of Micro/Nano Electronics, our team develop a glass fiber filter paper carrier, reaction box and color quantitative APP. We establish a mathematical model which reveals the relationship between the concentration
                        and color depth and it successfully fitted the experimental data. When in use, the user only needs to drop the sample onto the filter paper, and after colorizing, it can photograph by cell phone and analyze by the professional
                        APP to quantitatively detect the multi-factor concentration. This detection system is cheap, portable and has multiple applications, so it’s applicable to people's daily life testing. </p>
                    <p>
                        <a target="_blank" href="http://2017.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
                   <div class="section-helper" id="section11">
                    <hr>
                </div>
                <div class="onesection">
                    <h3 class="h3-responsive">2018 Colorectal cancer molecular detection</h3>
                                    <p>Colorectal cancer is a common type of malignant tumor worldwide, and with a morbidity rising sharply in China. Though early detection and treatment would effectively prevent the development of advanced colorectal cancer, the noncompliance with endoscopic screening, as well as the inability of methods alike to detect cancer at the molecular level keeps many patients from being diagnosed in time.
                    </p><p>Therefore, we aim to develop a non-invasive means to molecularly detect and locate the foci of colorectal-associated diseases, including colorectal cancer and colitis (which is a risk factor of cancer), using ultrasound. We plan to engineer bacteria that target colorectal cancer or respond to colitis, while producing gas vesicles to be detected by ultrasound. In this way, we hope to improve the screening method of colorectal cancer or pre-cancerous changes.
                                        
                    </p>
                    <p><a target="_blank" href="http://2018.igem.org/Team:SJTU-BioX-Shanghai">　　<i class="fa fa-angle-right" aria-hidden="true"></i> Wiki Page </a>
                    </p>
                </div>
            </div>

            <div class="col-md-4" id="coll">
                <div class="scroll-box grey lighten-5">
                    <nav class="list-unstyled" id="navbar-right" role="navigation">
                        <li class="nav-item">
                            <a class="nav-link" href="#section2">2009 E.coli the Napper</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section3">2010 Synthetic-biological Approaches to Osteoarthritis</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section4">2011 Codon Switch Controlling Protein Biosynthesis</a>
                        </li>
                         <li class="nav-item">
                            <a class="nav-link" href="#section5">2012 Membrane Magic</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section6">2013 Metabolic Gear Box</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section7">2014 Crown</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section8">2015 Cyano Pac-man</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section9">2016 Vesatile Yeast Interface - for disease diagnosis</a>
                        </li>
                        <li class="nav-item">
                            <a class="nav-link" href="#section10">2017 Palette</a>
                        </li>
                                                <li class="nav-item">
                            <a class="nav-link" href="#section11">2018 Colorectal cancer</a>
                        </li>
                    </nav>
                </div>
            </div>
        </div>

    </div>
    </div>

    <div class="old-browser-warning">
        <center>您使用的是过时的（不被支持的）浏览器，本网页可能无法正常展现。</center>
        <center>You are using an outdated (unsupported) browser and this page may not display properly.</center>
    </div>

    <footer class="page-footer center-on-small-only" style="padding-top:0">
        <div class="footer-copyright">
            <div class="container-fluid">
                <p><a href="./copyright-en.html">Copyright Details</a> © 2017 SJTU-BioX-Shanghai</p>
            </div>
        </div>
    </footer>

    <script>
        window.onload = function() {
            $(".scroll-box").css("width", $("#coll").width() - 30);
        };
    </script>
    <script src="./js/jquery-3.1.1.min.js"></script>
    <script src="./js/popper.min.js"></script>
    <script src="./js/bootstrap.min.js"></script>
    <script src="./js/mdb.min.js"></script>
    <script src="./js/countUp.js"></script>
    <script src="./js/site.js"></script>

</body>

</html>
